src/app/clusters/temperature-measurement-server/temperature-measurement-server.h

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/util/af-types.h>
+
+/** @brief Temperature Measurement Cluster Server Set Value
+ *
+ * Update the Temperature Measure Value
+ *
+ * @param endpoint   Ver.: always
+ * @param value   Ver.: always
+ *
+ */
+EmberAfStatus emberAfPluginTemperatureMeasurementSetValueCallback(chip::EndpointId endpoint, int16_t value);
+
+/** @brief Temperature Measurement Cluster Server Post Init
+ *
+ * Following resolution of the Temperature Measurement reading at startup for this endpoint,
+ * perform any additional initialization needed; e.g., synchronize hardware
+ * state.
+ */
+void emberAfPluginTemperatureMeasurementServerInitCallback(void);
```

Comment:
This is a bit weird but this one is already declared here: https://github.com/project-chip/connectedhomeip/blob/8c93ec6b67f9c37cfeec9fa1d2cb93266eb4aa95/src/app/util/util.cpp#L124

Reply:
Should I update the util file to consume this new header?

Reply:
Removed the definition in the header file, since Util seems to do its own weird thing here.